src/dev/build/tasks/os_packages/create_os_package_tasks.test.ts

```diff
@@ -121,6 +121,22 @@ it('runs task with the expected values for DEB ARM64', async () => {
             "optional",
           ],
         ],
+        Array [
```

Comment:
@sayuree This snapshot file change should be removed, as its unrelated to the change.